### ISSUE TITLE
Add a ColorInput field for custom colors to the TagPicker

### ISSUE THREAD

**Issue by SamuABC** [brendel-group/mission_db#128]

**Comment by pascal260303** [comment]
Wouldn't this be useful:
https://mantine.dev/core/color-picker/

**Comment by SamuABC** [comment]
> Wouldn't this be useful:
> 
> https://mantine.dev/core/color-picker/

Yes, thank you. I already know about the Mantine Color Picker. I am also using the Mantine ColorInput for the implementation of this issue :)
